lib/ex338/fantasy_team/standings.ex

```diff
@@ -56,4 +58,57 @@ defmodule Ex338.FantasyTeam.Standings do
 
     teams
   end
+
+  def add_season_ended_for_league(teams) do
+    Enum.map(teams, &(add_season_ended(&1)))
+  end
+
+  def add_season_ended(%{roster_positions: positions} = team) do
+    positions = calculate_season_ended(positions)
+
+    team
+    |> Map.delete(:roster_positions)
+    |> Map.put(:roster_positions, positions)
+  end
+
+  defp calculate_season_ended(positions) do
+    Enum.map(positions, &(verify_and_add_info(&1)))
+  end
+
+  defp verify_and_add_info(
+    %{fantasy_player: %{sports_league: %{championships: []}}} = position) do
+      Map.put(position, :season_ended?, false)
+  end
+
+  defp verify_and_add_info(
+    %{fantasy_player: %{sports_league: %{championships: overall}}} = position) do
+
+      season_ended? =
```

Comment:
Assigned variable `season_ended?` has same name as a private function in the same module.